app/uk/gov/hmrc/agentoverseasfrontend/views/application/main_business_address.scala.html

```diff
@@ -47,50 +47,51 @@
+                </h1>
+                <div class="govuk-body">@Messages("mainBusinessAddress.body.text")</div>
+            </legend>
+
+            @govukInput(Input(
+                label = Label(content = Text(Messages("mainBusinessAddress.form.addressLine1"))),
+                spellcheck = Some(false),
+                classes = "govuk-input--width-20"
+            ).withFormField(mainBusinessAddressForm("addressLine1")))
 
-      @govukInput(Input(
-          label = Label(content = Text(Messages("mainBusinessAddress.form.addressLine2"))),
-          spellcheck = Some(false),
-          classes = "govuk-input--width-20",
-          inputType = "text",
-      ).withFormField(mainBusinessAddressForm("addressLine2")))
+            @govukInput(Input(
+                label = Label(content = Text(Messages("mainBusinessAddress.form.addressLine2"))),
+                spellcheck = Some(false),
+                classes = "govuk-input--width-20"
+            ).withFormField(mainBusinessAddressForm("addressLine2")))
 
-      @govukInput(Input(
-          label = Label(content = Text(Messages("mainBusinessAddress.form.addressLine3"))),
-          spellcheck = Some(false),
-          classes = "govuk-input--width-20",
-          inputType = "text",
-      ).withFormField(mainBusinessAddressForm("addressLine3")))
+            @govukInput(Input(
+                label = Label(content = Text(Messages("mainBusinessAddress.form.addressLine3"))),
+                spellcheck = Some(false),
+                classes = "govuk-input--width-20"
+            ).withFormField(mainBusinessAddressForm("addressLine3")))
 
-      @govukInput(Input(
-          label = Label(content = Text(Messages("mainBusinessAddress.form.addressLine4"))),
-          spellcheck = Some(false),
-          classes = "govuk-input--width-20",
-          inputType = "text",
-      ).withFormField(mainBusinessAddressForm("addressLine4")))
+            @govukInput(Input(
+                label = Label(content = Text(Messages("mainBusinessAddress.form.addressLine4"))),
+                spellcheck = Some(false),
+                classes = "govuk-input--width-20"
+            ).withFormField(mainBusinessAddressForm("addressLine4")))
 
-      <div class="autocomplete-wrapper" id="countryCode">
-        @govukSelect(Select(
-            label = Label(content = Text(Messages("mainBusinessAddress.form.country"))),
-            id = "countryCode-auto-complete",
-            name = "countryCode",
-            items = SelectItem(value = Some("")) :: countries.toList.sortBy(_._2).map { case (code, name) => SelectItem(value = Some(code), text = name) }
-        ).withFormField(mainBusinessAddressForm("countryCode")))
-      </div>
+            <div class="autocomplete-wrapper" id="countryCode">
+                @govukSelect(Select(
+                    label = Label(content = Text(Messages("mainBusinessAddress.form.country"))),
+                    id = "countryCode-auto-complete",
```

Comment:
I just realised in the javascript the id is still set to amlsCode - change it to "countryCode-auto-complete" and remove the autocomplete wrapper div since that is created by the JS. If there is also an amls autocomplete in the project, you will need a JS enhancement for each of them since they have different id's & options.
```
var selectEl = document.querySelector('#amlsCode');
    if(selectEl){
        accessibleAutocomplete.enhanceSelectElement({
            autoselect: true,
            defaultValue: selectEl.options[selectEl.options.selectedIndex].innerHTML,
            selectElement: selectEl
        })
    }
```